model-optimizer/extensions/back/ReverseInputChannels.py

```diff
@@ -50,10 +50,31 @@ class InsertReverseChannels(BackReplacementPattern):
     """
     enabled = False
 
+    @staticmethod
+    def get_fw_index(node: Node, idx: int):
+        if not node.has_valid('rt_info'):
+            return idx
+
+        rt_info = node.rt_info
+        if not rt_info.contains('old_api_map'):
+            return idx
+
+        old_api_map_version = rt_info.get_attribute_version('old_api_map')
+        old_api_map = rt_info.info['old_api_map', old_api_map_version]
+        if 'inverse_order' not in old_api_map.info:
+            return idx
+
+        order = old_api_map.info['inverse_order']
+        node_name = node.soft_get('name', node.id)
+        assert len(order) > idx >= 0, \
```

Comment:
Can we please make it work for negative axis too?

There is one particular transformation which inserts ReverseInputChannels operator outside of this file -- [here](https://github.com/openvinotoolkit/openvino/blob/master/model-optimizer/extensions/front/tf/UnpackPackReverseInputChannels.py#L67)

And it is indeed possible to have negative axis from there due to 
TF docs [here](https://www.tensorflow.org/api_docs/python/tf/unstack) and [here](https://www.tensorflow.org/api_docs/python/tf/stack) 

Could you please add such a unit-test?

Reply:
Done.